watson_developer_cloud/nlu/features/v1/__init__.py

```diff
@@ -0,0 +1,72 @@
+class Feature(object):
```

Comment:
Not sure now this works -- does it essentially type-check the valid keys in the dict they send in the features parameter?